pkg/addon/configpolicy/agent_addon.go

```diff
@@ -105,6 +107,9 @@ func getValues(cluster *clusterv1.ManagedCluster,
 	// Enable Prometheus metrics by default on OpenShift
 	userValues.Prometheus["enabled"] = userValues.KubernetesDistribution == "OpenShift"
 
+	// Disable OperatorPolicy if the cluster is not on OpenShift version 4.y
+	userValues.OperatorPolicy["disabled"] = cluster.Labels["openshiftVersion-major"] != "4"
+
```

Comment:
With `mandateValues()`, it can be configurable for OCP 4 users and nobody else.

Okay, I was also thinking once OCP 5 rolled around, we'd theoretically have an integration test that would start failing, so we'd have to handle it pretty directly.